ETWAnalyzer_uTest/EventDump/DumpFileTests.cs

```diff
@@ -37,9 +36,63 @@ public void DumpFileCheck()
                 FileRenameCount = 3,
                 FileOpenCloseTimeInus = 201,
             };
+
+            dump.SortOrder = ETWAnalyzer.Commands.DumpCommand.SortOrders.OpenCloseTime;
             decimal totalTime = dump.GetSortValue(groupedData);
             Assert.Equal(201, totalTime);
 
+            dump.SortOrder = ETWAnalyzer.Commands.DumpCommand.SortOrders.Length;
```

Comment:
The documented sort orders are:
 Count (Open+Close+Read+Write+SetSecurity)
 ReadSize
 WriteSize
 ReadTime
 WriteTime
 TotalSize 
 TotalTime (= Open+Close+Read+Write). 

Size, Time, Length are very generic and although there is some sorting defined it is unclear what it should be. 
Normally one would sort by the documented ones first.

Reply:
fixed